CHANGES

```diff
@@ -156,7 +156,8 @@ Universes
 - Cumulative inductive types. see prefixes "Cumulative", "NonCumulative"
   for inductive definitions and the option "Set Inductive Cumulativity"
   in the reference manual.
-- New syntax `Type@{_}` for anonymous universes.
+- New syntax `Type@{_}` for anonymous universes (can serve to instantiate
+  explicitly a polymorphic definition with a flexible universe).
```

Comment:
@SkySkimmer Is that better? Maybe the mention of "flexible universe" is still too much and the CHANGES entry should stop at "definition"?